MAINTAINERS

```diff
@@ -1 +1,10 @@
-Jimmi Dyson <jimmidyson@gmail.com> (@jimmidyson)
+Jimmi Dyson           <jimmidyson@gmail.com>        ( @jimmidyson )
+Hardy Ferentschik                                   ( @hferentschik )
```

Comment:
@hferentschik really remove? people can still track it down to be

```
Author: Hardy Ferentschik <hardy@hibernate.org>
```

Aesthetically it looks weird now... ... really, Hardy? just add... ;-)